napari/utils/colormaps/colormap_utils.py

```diff
@@ -370,28 +370,30 @@ def label_colormap(num_colors=256, seed=0.5):
     ----------
     num_colors : int, optional
         Number of unique colors to use. Default used if not given.
+        Colors are in addition to a transparent color 0.
     seed : float or array of float, length 3
         The seed for the random color generator.
 
     Returns
     -------
     colormap : napari.utils.Colormap
-        A colormap for use with labels are remapped to [0, 1].
+        A colormap for use with labels remapped to [0, 1].
 
     Notes
     -----
     0 always maps to fully transparent.
     """
     # Starting the control points slightly above 0 and below 1 is necessary
     # to ensure that the background pixel 0 is transparent
-    midpoints = np.linspace(0.00001, 1 - 0.00001, num_colors - 1)
+    midpoints = np.linspace(0.00001, 1 - 0.00001, num_colors)
     control_points = np.concatenate(([0], midpoints, [1.0]))
     # make sure to add an alpha channel to the colors
     colors = np.concatenate(
         (_color_random(num_colors, seed=seed), np.full((num_colors, 1), 1)),
         axis=1,
     )
-    colors[0, :] = 0  # ensure alpha is 0 for label 0
+    # Insert alpha at layer 0
+    colors = np.concatenate((np.full((1, 4), 0.0), colors))
```

Comment:
Prepending instead of replacing means that the first color of the default label colormap changes, right? From scratch, I don't see anything necessarily wrong with that, but the experience might be quite jarring for an experience napari user. @jni : I'd be interested in your opinion here.

Reply:
On second thought, there is no reason to muddy the waters. I'll revert to the original method (but with an extra color at the end.)